src/aggregate/aggregate_exec.c

```diff
@@ -51,8 +52,17 @@ void AggregateCommand_ExecAggregateEx(RedisModuleCtx *ctx, RedisModuleString **a
     IndexLoadOptions lOpts = {.name = {.cstring = settings->cursorLookupName}};
     IndexSpec *sp = IndexSpec_LoadEx(ctx, &lOpts);
     if (!sp) {
-      RedisModule_ReplyWithError(ctx, "Unknown Index name");
-      return;
+      // check if its not alias
```

Comment:
This probably makes more sense to do in the coordinator... but since it's 1.4, I guess it should work here without issue as well.